shapely/plotting.py

```diff
@@ -9,6 +9,12 @@
 
 import shapely
 
+__all__ = [
+    "plot_polygon",
```

Comment:
```suggestion
    "patch_from_polygon",
    "plot_polygon",
```

with the other PR merged, we can add this here